asn1ate/sema.py

```diff
@@ -719,6 +721,18 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self,elements):
+	self.module2symbols = { }
+	for i in range (0,len(elements),2):
+		self.module2symbols [elements [i+1].elements[0]] = set (elements [i])
+	print 'Imported from', self.module2symbols.keys ()
+
+class Exports(Imports):
```

Comment:
This looks assym,etric -- can you add a third sema node, say, `ModuleSymbols` and have both `Imports` and `Exports` derive from it?